src/scenes/UserGalleryPage/QRCode.tsx

```diff
@@ -0,0 +1,207 @@
+import colors from 'components/core/colors';
+import styled from 'styled-components';
+import { useCallback, useState, useRef } from 'react';
+import Spacer from 'components/core/Spacer/Spacer';
+// import QRCodeStyling from 'qr-code-styling';
+
+function CloseIcon() {
+  return (
+    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M12.6667 3.33334L3.33333 12.6667" stroke="black" stroke-miterlimit="10" />
+      <path d="M3.33333 3.33334L12.6667 12.6667" stroke="black" stroke-miterlimit="10" />
+    </svg>
+  );
+}
+
+function QRIcon() {
```

Comment:
another good candidate to move to `src/icons` - in case we want to reuse somewhere else in the future